index.d.ts

```diff
@@ -113,6 +114,7 @@ declare namespace BeeQueue {
     on(ev: 'retrying', fn: (err: Error) => void): this;
     on(ev: 'failed', fn: (err: Error) => void): this;
     on(ev: 'progress', fn: (progress: any) => void): this;
+    on(ev: 'removed', fn: () => void): this;
```

Comment:
Needs to be documented in the readme. Add a code example showing how jobs should handle their 'removed' event internally. Note that it requires the 'getEvents' option.

Reply:
I added some documentation in the readme